core/src/main/resources/AppConstants.properties

```diff
@@ -352,6 +352,9 @@ etag.ehcache.dir=${cache.dir}
 etag.ehcache.diskPersistent=false
 etag.ehcache.diskExpiryThreadIntervalSeconds=600
 
+########## OLD CONSOLE ##########
+legacyConsole.enabled=false
```

Comment:
Niet 'old' en 'legacy' noemen, ook geen 'ugly' of 'beautiful', maar naar iets wat het is. Zoiets als 'struts console'.